pkg/apis/settings/settings.go

```diff
@@ -34,8 +34,10 @@ type settingsKeyType struct{}
 var ContextKey = settingsKeyType{}
 
 var defaultSettings = &Settings{
+	AssumeRole:                 "",
 	ClusterName:                "",
 	ClusterEndpoint:            "",
+	CredsDuration:              15,
```

Comment:
We typically avoid abbr va na 
```suggestion
	CredentialsDuration:              15,
```

Reply:
Fixed in next revision with renaming of settings from 'credsDuration' to 'assumeRoleDuration'

Reply:
It's really the duration of a _session_. See https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp_use-resources.html for context.

Reply:
I decided to stick with `assumeRoleDuration` since `assumeRoleSessionDuration` is a very long name that I don't believe adds enough information to justify the impact in readability.